### DISCUSSION TITLE
Hvilken måler benytter du?

### DISCUSSION THREAD

**Discussion by toreamun** [toreamun/amshan-homeassistant#6] [General]
Jeg ønsker å kartlegge hvilke målere som benyttes med denne integrasjonen. Den er testet med testdata for Kaifa, Aidon og Kamstrup, og selv kjører jeg den for Aidon 6525 med listeversjon AIDON_V0001 og Kaifa MA304H3E med listeversjon KFM_001. Det kan være spesielt interessant å vite listeversjon (står som firmware under device info) og om du benytter mqtt (inklusiv mqtt kilde, f.eks Tibber).

Det er typisk det som står under integrasjonens enhetsinformasjon som er av interesse:
![image](https://user-images.githubusercontent.com/12134766/146072502-0e61baf5-c8d9-4f83-b063-eaf774783124.png)

Mine målere er:
- Aidon, 6525, AIDON_V0001
- Kaifa, MA304H3E, KFM_001

## Oversikt
| Fabrikat | Type | Listeversjon |  | oppkobling | 
| ----------- | ----------- |----------- |-----------|----------- |
| Aidon | 6515 | AIDON_V0001 | @ctvedt  | tcp/ip |
| Aidon | 6525 | AIDON_V0001 | @toreamun, aTommyC  | serieport, tcp/ip, mqtt (Tibber Pulse) |
| Aidon | 6540 | AIDON_V0001 | @mnordseth | |
| Kaifa | MA304H3E | KFM_001 | @toreamun | serieport, tcp/ip, mqtt |
| Kaifa | MA105H2E | KFM_001 | @ctvedt | |
| Kamstrup | 6861111BN242101040 | Kamstrup_V0001 | | mqtt (Tibber Pulse) |
| Landis+Gyr | E360 | P1 | @koppelo | mqtt (Tibber Pulse) |

**Comment by ctvedt** [comment]
Kaifa, MA105H2E, KFM_001
og
Aidon, 6515, AIDON_V0001

**Reply by ctvedt** [reply]
@toreamun ser jeg er listet opp under 6525, og ikke 6515. Forøvrig bruker jeg ser2net på begge mine målere.

**Reply by toreamun** [reply]
Beklager. Fikse nå ;-)

**Comment by mnordseth** [comment]
Aidon 6540 med listeversjon AIDON_V0001 her.

**Comment by aTommyC** [comment]
Aidon 6525 AIDON_V0001

**Comment by agekristian** [comment]
hei
er det noen som har fått Kamstrup til å virke med AMS reader ?
får bare "Timeout reading data"

**Reply by toreamun** [reply]
Med AMS reader, mener du da https://github.com/Danielhiversen/AMSreader?

**Reply by StarkJohan** [reply]
Jag har en Kamstrup och har med ESP8266 + MBusmodul från Kina fått igång AmsToMqttBridge utan problem.

**Reply by StarkJohan** [reply]
Kamstrup, Kamstrup_V0001, 6841138AN243101090
Har nu Wemos D1 Mini direktkopplat i min mätare, alltså utan HAN eller P1-modul. Serial_8N1 och 2400 baud. 
Pinout: [https://xipher.dk/assets/images/pinout.png](url)
Integrationen fungerar fint med rådata (bytes, inte hex) över MQTT.

**Comment by agekristian** [comment]
Nei, jeg mente denne : https://github.com/gskjold/AmsToMqttBridge

**Reply by toreamun** [reply]
https://github.com/gskjold/AmsToMqttBridge har planlagt støtte for å videresende de nødvendige "rå" meldinger fra måleren fra versjon 2.1. I [planen](https://github.com/gskjold/AmsToMqttBridge/milestone/22?closed=1) står [oppgaven](https://github.com/gskjold/AmsToMqttBridge/issues/210) som lukket allerede, så dette går nok som planlagt.

**Comment by tfriberg** [comment]
Landis+Gyr E360 (P1/HAN/RJ12) MQTT via Tibber Pulse

**Comment by douglas2** [comment]
Fungerar utmärkt med Aidon 6534 (Jönköpings Energi) med USB porten!

Hittar tyvärr inte vilken version/firmware jag har. Står bara så här:
Device info
6534
by ADN

![5CD56961-7892-47F5-87DE-F2C3AF823683](https://user-images.githubusercontent.com/186145/190243499-c729a4b7-2af7-48b1-a714-a123771144b4.jpeg)

Använder denna:
P1/HAN till USB-adapter:
https://energyintelligence.se/shop/product/p1-han-till-usb-adapter

**Comment by ingolfzp** [comment]
Jeg har en Kamstrup-måler (HAN) + Tibber Pulse i lokal MQTT-modus.
Den ble detektert som Kaifa:
![2023-05-05 08_14_39-Window](https://user-images.githubusercontent.com/45397048/236388631-c65f6808-b596-4fd5-b2f6-5f87226b19d4.png)

Ser også at Volt på L2 mangler (dette var data jeg fikk ut når Tibber Pulse var koblet via Tibber-skyen):

![2023-05-05 08_17_08-Window](https://user-images.githubusercontent.com/45397048/236388810-7986ea83-f7f1-4c17-922a-5076dd02b464.png)

**Comment by Takst3in** [comment]
@ingolfzp fant du ut av problem med manglende Volt på L2?

**Reply by ingolfzp** [reply]
Dessverre ikke. Samme status her.
Kan nevne at integrasjonen ellers har fungert prikkfritt, så L2-Volt hadde jo gjort den perfekt!

**Reply by Takst3in** [reply]
Ok.. Meldte en egen ticket på dette i går kveld, satser på at vi får den inn etter hvert da.. 👍

**Comment by lindsand** [comment]
Energyintelligences P1-läsare fungerar också i Finland, samma standard
https://sesko.fi/wp-content/uploads/2022/12/Recommendation-SK-13-1_H1_customer-interface_englanninkielinen_2022-12-13.pdf

Jag har kört den här tillsammans med en **AIDON 7534** nu i 1/2 år problemfritt:
https://energyintelligence.se/shop/product/p1-han-lasare-ethernet-poe-wifi-extern-antenn

Så README.md kan uppdateras för FI:

https://github.com/toreamun/amshan-homeassistant/blob/master/README.md
![image](https://github.com/toreamun/amshan-homeassistant/assets/66431079/097568ca-61cc-48cd-a574-713cc71928c0)

Message 0 received on Aidon_7534/raw at 20:23:
/ADN9 7534

0-0:1.0.0(231107202300W)
1-0:1.8.0(00002724.836*kWh)
1-0:2.8.0(00002007.983*kWh)
1-0:3.8.0(00000036.114*kVArh)
1-0:4.8.0(00000895.320*kVArh)
1-0:1.7.0(0000.972*kW)
1-0:2.7.0(0000.000*kW)
1-0:3.7.0(0000.000*kVAr)
1-0:4.7.0(0000.430*kVAr)
1-0:21.7.0(0000.016*kW)
1-0:22.7.0(0000.000*kW)
1-0:41.7.0(0000.870*kW)
1-0:42.7.0(0000.000*kW)
1-0:61.7.0(0000.080*kW)
1-0:62.7.0(0000.000*kW)
1-0:23.7.0(0000.000*kVAr)
1-0:24.7.0(0000.164*kVAr)
1-0:43.7.0(0000.000*kVAr)
1-0:44.7.0(0000.132*kVAr)
1-0:63.7.0(0000.000*kVAr)
1-0:64.7.0(0000.140*kVAr)
1-0:32.7.0(230.3*V)
1-0:52.7.0(229.2*V)
1-0:72.7.0(230.5*V)
1-0:31.7.0(000.7*A)
1-0:51.7.0(003.8*A)
1-0:71.7.0(000.6*A)
!822E
QoS: 0 - Retain: false

![image](https://github.com/toreamun/amshan-homeassistant/assets/66431079/9b0075a7-4089-4817-9acb-d2076c1df44a)

**Comment by sindrej** [comment]
![image](https://github.com/toreamun/amshan-homeassistant/assets/1797266/c55a6352-7e6b-41db-82ab-782a718ee7a0)

Tibber Pulse (MQTT)
